docker-compose.yml

```diff
@@ -27,12 +27,12 @@ services:
   api:
     env_file:
       - .env
-    image: postgrest/postgrest
+    image: postgrest/postgrest:v0.4.4.0
```

Comment:
I'd rather remove this change from this particular PR because:

1. This has nothing to do with the PR title
2. Pinning the version is cool, but we need to document why `v0.4.4.0` instead of `v0.5.0.0` released 5 days ago – and this is why a PR dedicated to this issue can be helpful